custom/icds_reports/static/js/directives/program-summary/program-summary.directive.js

```diff
@@ -144,7 +144,9 @@ function ProgramSummaryController($scope, $http, $log, $routeParams, $location,
     });
     $scope.$on('mobile_filter_data_changed', function (event, data) {
         vm.filtersOpen = false;
-        if (data.hasLocation) {
+        if (!data.location) {
+            vm.moveToLocation('national', -1);
```

Comment:
might be nice to make this a constant, especially if it is used elsewhere

Reply:
agreed, though: 
![image](https://user-images.githubusercontent.com/66555/70135044-eec24700-1691-11ea-89b2-1d833f719544.png)

(will look for opportunities to improve these though)